devices/Vl6180X/samples/Vl6180x.sample/Program.cs

```diff
@@ -0,0 +1,29 @@
+using Iot.Device.VL6180X;
+using nanoFramework.Hardware.Esp32;
+using System;
+using System.Device.I2c;
+using System.Diagnostics;
+using System.Threading;
+
+namespace Vl6180x.sample
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            Debug.WriteLine("Hello VL6180X!");
+            // when connecting to an ESP32 device, need to configure the I2C GPIOs used for the bus
+            Configuration.SetPinFunction(11, DeviceFunction.I2C1_DATA);
+            Configuration.SetPinFunction(10, DeviceFunction.I2C1_CLOCK);
+
+            using VL6180X sensor = new(I2cDevice.Create(new I2cConnectionSettings(1, VL6180X.DefaultI2cAddress)));
+            sensor.Init();
+            while (true)
+            {
+                var distance = sensor.ReadRange();
+                Console.WriteLine($"Distance: {distance.Centimeters} cm.");
+                Thread.Sleep(500);
```

Comment:
maybe add a note that you always have to wait a bit before reading a new distance. Btw, you can create a public constant or equivalent with the minimal recommended time to read (we have this in couple of other bindings)

Reply:
Not sure if there is any sleep required. Added this one just to trim output in console.

Reply:
fair, it depends of the mode and seems that in continuous measurement, it does an averaging and you the always get a data. It may not be accurate, but you'll get something.

Reply:
Continuous measurement is not implemented. It should be event based with INT pin implemented - for now I don't need it in my project, thus I haven't implemented it.